source/digits_hits/include/GateRejectedVolume.hh

```diff
@@ -0,0 +1,67 @@
+/*----------------------
+   Copyright (C): OpenGATE Collaboration
+
+This software is distributed under the terms
+of the GNU Lesser General  Public Licence (LGPL)
+See LICENSE.md for further details
+----------------------*/
+
+
+#ifndef GateRejectedVolume_h
+#define GateRejectedVolume_h 1
+
+#include "globals.hh"
+#include <iostream>
+#include <vector>
+#include "G4ThreeVector.hh"
+
+#include "GateVPulseProcessor.hh"
+
+class GateRejectedVolumeMessenger;
+
+
+/*! \class  GateRejectedVolume
+    \brief  Pulse-processor modelling a simple volume discriminator.
+
+    - The method ProcessOnePulse of this class models a simple
+      volume discriminator: any input pulse whose volume name is not as
+      defined is copied into the output pulse-list.
+      On the contrary, any input pulse whose name is the same as 
+      specified is discarded.
+
+      \sa GateVPulseProcessor
+*/
+class GateRejectedVolume : public GateVPulseProcessor
+{
+  public:
+
+    //! Constructs a new module attached to a GateDigitizer
+    GateRejectedVolume(GatePulseProcessorChain* itsChain,
+			       const G4String& itsName, G4String rejectedVolume) ;
```

Comment:
It seems there is something wrong in formatting of all files. I couldn't find official formatting. There is also space before ``;``.

Reply:
I removed space